### ISSUE TITLE
Shadow not displayed on iOS

### ISSUE THREAD

**Issue by shameister** [Microsoft/reactxp#925]
When comparing my app on web (Electron) and iOS Simulator, I see, that the shadow property is not rendered on iOS.

How can this be solved?

![shadow issue](https://user-images.githubusercontent.com/98413/48635288-074cba00-e9c8-11e8-8f95-cefc481032cd.png)

*Style of the view*

```
RX.Styles.createViewStyle({
    flexDirection: 'column',
    flexGrow: 1,
    backgroundColor: Colors.WHITE,
    left: 0,
    right: 2,
    position: 'absolute',
    
    shadowColor: 'rgb(204, 204, 204),
    shadowRadius: 3,
    shadowOffset: { height: 2, width: 2 }
 }),
```

Looking forward to solve this.

Regards,
Sascha

**Comment by erictraut** [comment]
I think this is because the shadow rendering on iOS doesn't honor clipping, whereas the web does. You can test this theory by setting the overflow to 'visible', which should disable clipping for the element.

**Comment by erictraut** [comment]
Your app is looking really good, by the way!

**Comment by shameister** [comment]
> You can test this theory by setting the overflow to 'visible', which should disable clipping for the element. 

This is exactly, what I expected to be the solution. I already tried this. It does not change the behavior.

What can I try or provide for further investigation?

> Your app is looking really good, by the way!

Thanks :-)

**Comment by erictraut** [comment]
Can you view it in the Chrome debugger and play with the styles in the DOM? That's probably the easiest way to debug it.

**Comment by shameister** [comment]
Hey Eric,

thanks for the idea. I haven't used the Chrome debugger so far. As I tried, I could not spot any mistake. Maybe you can?

I colored the container orange, so it can be better spotted.
![bildschirmfoto 2018-11-19 um 11 12 06](https://user-images.githubusercontent.com/98413/48700496-6854d780-ebec-11e8-9dc7-50438d87d816.png)

This is the view, that displays shadows properly on web (Electron).

![bildschirmfoto 2018-11-19 um 11 15 44](https://user-images.githubusercontent.com/98413/48700567-9508ef00-ebec-11e8-8a9c-d7446a62b725.png)

**Comment by shameister** [comment]
I also like to add, that `shadowOpacity is` [missing in the docs](https://microsoft.github.io/reactxp/docs/styles.html).

**Comment by erictraut** [comment]
Yes, you're correct. It's missing in the docs and in the test app. 

On RN, shadowOpacity defaults to 1, so that should be the documented behavior for ReactXP. We can add code to web/Styles.ts that sets the default value if it sees shadowOffset, shadowRadius, or shadowColor set but no shadowOpacity value specified.

I'm not sure when I'll have time to do this. If you have time and are so inclined, feel free to create a PR that addresses the above issues.

Thanks for reporting the problem.

**Comment by fbartho** [comment]
I've verified that this is still a necessary documentation update! Separately re: the Feature, @erictraut if you want to share a hint of how/where in Styles.ts to add this default-setting code, I'd be happy to implement it!
